openwisp_radius/tasks.py

```diff
@@ -43,9 +43,9 @@ def deactivate_expired_users():
 
 
 @shared_task
-def delete_old_radiusbatch_users(older_than_months=12):
+def delete_old_radiusbatch_users(older_than_days=365):
```

Comment:
shouldn't the default here be the same as `BATCH_DELETE_EXPIRED`? This was probably a mistake.

Reply:
Sure we can change this